tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -955,7 +955,7 @@ MetroHash::Hash PipelineDumper::GenerateHashForComputePipeline(
 // Updates hash code context for vertex input state
 void PipelineDumper::UpdateHashForVertexInputState(
     const VkPipelineVertexInputStateCreateInfo* pVertexInput,  // [in] Vertex input state
-    MetroHash64*                                pHasher)       // [in,out] Haher to generate hash code
+    MetroHash128*                                pHasher)       // [in,out] Haher to generate hash code
```

Comment:
The alignment in this file and in ` tool/dumper/vkgcPipelineDumper.h` should be updated :)